addon/components/polaris-popover/content.js

```diff
@@ -82,29 +81,40 @@ export default Component.extend({
       return;
     }
 
-    const trigger = Ember$(
+    const trigger = document.querySelector(
       `div.ember-basic-dropdown-trigger[data-ebd-id="${uniqueId}-trigger"]`
-    )[0];
-    const content = Ember$(`div#ember-basic-dropdown-content-${uniqueId}`)[0];
+    );
+    const content = document.querySelector(
+      `div#ember-basic-dropdown-content-${uniqueId}`
+    );
 
     if (isNone(trigger) || isNone(content)) {
       return;
     }
 
     const triggerRect = getRectForNode(trigger);
-    const left =
-      triggerRect.width / 2 + (triggerRect.left - getRectForNode(content).left);
-    Ember$('div.Polaris-Popover__Tip', content).css({ left });
+    const left = `${triggerRect.width / 2 +
+      (triggerRect.left - getRectForNode(content).left)}px`;
+    const toolTip = document.querySelector('div.Polaris-Popover__Tip');
+
+    if (toolTip) toolTip.style.left = left;
```

Comment:
Can we avoid single-line, un-curlied `if`-statements please? 🙏

Reply:
Sure 😝